ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/ExceptionRequestDetailsPage.tsx

```diff
@@ -0,0 +1,175 @@
+            return `Approved pending update (${exception.cves.length} CVEs)`;
+        default:
+            return '';
+    }
+}
+
+function ExceptionRequestDetailsPage() {
+    const { requestId } = useParams();
+    const vulnerabilityException = vulnerabilityExceptions.find(
+        (exception) => exception.id === requestId
+    );
+
+    if (!vulnerabilityException) {
+        return (
+            <NotFoundMessage
+                title="404: We couldn't find that page"
+                message={`A request with ID ${requestId as string} could not be found.`}
+            />
+        );
+    }
+
+    // There will always be at least 1 comment
+    const latestComment =
+        vulnerabilityException.comments[vulnerabilityException.comments.length - 1];
+
+    return (
+        <>
+            <PageSection variant="light" className="pf-u-py-md">
+                <Breadcrumb>
+                    <BreadcrumbItemLink to={exceptionManagementPath}>
+                        Exception management
+                    </BreadcrumbItemLink>
+                    <BreadcrumbItem isActive>{vulnerabilityException.name}</BreadcrumbItem>
+                </Breadcrumb>
+            </PageSection>
+            <Divider component="div" />
+            <PageSection
+                className="pf-u-display-flex pf-u-flex-direction-row pf-u-align-items-center"
+                variant="light"
+            >
+                <Flex direction={{ default: 'column' }}>
+                    <Title headingLevel="h1">Request {vulnerabilityException.name}</Title>
+                    <FlexItem>{getSubtitleText(vulnerabilityException)}</FlexItem>
+                </Flex>
+            </PageSection>
+            <PageSection>
+                <PageSection variant="light">
+                    <Flex direction={{ default: 'column' }}>
+                        <FlexItem>
+                            <Title headingLevel="h2">Overview</Title>
+                        </FlexItem>
+                        <FlexItem flex={{ default: 'flexNone' }}>
+                            <DescriptionList className="vulnerability-exception-request-overview">
+                                <DescriptionListGroup>
+                                    <DescriptionListTerm>Requestor</DescriptionListTerm>
+                                    <DescriptionListDescription>
+                                        {vulnerabilityException.requester.name}
+                                    </DescriptionListDescription>
+                                </DescriptionListGroup>
+                                <DescriptionListGroup>
+                                    <DescriptionListTerm>Requested action</DescriptionListTerm>
+                                    <DescriptionListDescription>
+                                        <RequestedAction
+                                            exception={vulnerabilityException}
+                                            context="PENDING_REQUESTS"
```

Comment:
Let's not worry about that right now. This was basically what I asked to UX earlier in this message https://redhat-internal.slack.com/archives/C03JMGWJYMD/p1698955614283599. I'm waiting for a reply to that. I'll bring it up again in today's meeting. Since we're showing the same request data regardless of which context the user came from (ie. Pending Requests or Approved deferrals/false positives) we need a smarter way to distinguish the original request expiry/cves from the updated one. For now, I just set it to this. I'll add a comment to make it clearer